python/modules/Stop0lBaselineProducer.py

```diff
@@ -220,7 +225,7 @@ def analyze(self, event):
         ## In case JEC changed jet pt order, resort jets
         sortedPhi = self.GetJetSortedIdx(jets)
         PassdPhiLowDM   = self.PassdPhi(sortedPhi, [0.5, 0.15, 0.15])
-	PassdPhiMedDM   = self.PassdPhi(sortedPhi, [0.15, 0.15, 0.15], invertdPhi=True) #Variable for LowDM Validation bins
+	PassdPhiMedDM   = self.PassdPhiVal(sortedPhi, [0.15, 0.15, 0.15], [0.5, 4., 4.]) #Variable for LowDM Validation bins
```

Comment:
But this is not the medDM definition from Hui. Can @wang-hui confirm this? How about the meddphi for highDM?

Reply:
The Low DM Validation region is 0.15 < dPhi1 < 0.5 && dPhi2 > 0.15 and dPhi3 (if exists) > 0.15.
So this is correct.

Reply:
I wrote it out explicitely.

Reply:
Ok, shound't max dPhi could pi - (-pi) = 2pi? Why 4. only?